### PR TITLE
fix description link

### PR DESCRIPTION
**Pull request by hugofonseca**
I didn't test this because I don't have the environment configured but I think that this does not need that though.
The thing is taht I have this in my console
```<a href=https://bitbucket.org/org_duimmyy/projectdummy/pull-request/39>#39 PROJECT-1368</a>```
And I think that my PR will fix this.

Also, I don't know id this is the right place to PR. please tell me if I'm mistaken @nishio-dens

### CODE REVIEW

File: src/main/java/bitbucketpullrequestbuilder/bitbucketpullrequestbuilder/BitbucketCause.java [hunk 1]
```diff
@@ -83,9 +83,9 @@ public String getPullRequestTitle() {
 
     @Override
     public String getShortDescription() {
-        String description = "<a href=" + BITBUCKET_URL + this.getDestinationRepositoryOwner() + "/";
+        String description = '<a href="' + BITBUCKET_URL + this.getDestinationRepositoryOwner() + "/";
         description += this.getDestinationRepositoryName() + "/pull-request/" + this.getPullRequestId();
-        description += ">#" + this.getPullRequestId() + " " + this.getPullRequestTitle() + "</a>";
+        description += '">#' + this.getPullRequestId() + " " + this.getPullRequestTitle() + "</a>";
         return description;
```
Comment (nishio-dens) [inline src/main/java/bitbucketpullrequestbuilder/bitbucketpullrequestbuilder/BitbucketCause.java]: Thank you for your PR.
But I cannot compile it. 
Could you fix it like this?
```
--- a/src/main/java/bitbucketpullrequestbuilder/bitbucketpullrequestbuilder/BitbucketCause.java
+++ b/src/main/java/bitbucketpullrequestbuilder/bitbucketpullrequestbuilder/BitbucketCause.java
@@ -83,9 +83,9 @@ public class BitbucketCause extends Cause {

     @Override
     public String getShortDescription() {
-        String description = '<a href="' + BITBUCKET_URL + this.getDestinationRepositoryOwner() + "/";
+        String description = "<a href=\"" + BITBUCKET_URL + this.getDestinationRepositoryOwner() + "/";
         description += this.getDestinationRepositoryName() + "/pull-request/" + this.getPullRequestId();
-        description += '">#' + this.getPullRequestId() + " " + this.getPullRequestTitle() + "</a>";
+        description += "\">#" + this.getPullRequestId() + " " + this.getPullRequestTitle() + "</a>";
         return description;
     }
 }
```

**Comment by hugofonseca** [comment]
sorry about that, do you want me to squash this?

**Comment by nishio-dens** [comment]
GJ! Thanks!

**Comment by hugofonseca** [comment]
Thank you, can you please publish a new release of the hpi?

**Comment by hugofonseca** [comment]
I mean create a new release.

**Pull request merged**